### ISSUE TITLE
Copy does not work

### ISSUE THREAD

**Issue by cboelter** [MetaModels/core#743]
If I want to copy an item I get the following screen and nothing happens:

![bildschirmfoto 2015-02-05 um 10 55 15](https://cloud.githubusercontent.com/assets/550207/6058201/7f34c0b4-ad25-11e4-80d8-b4ecac9f9fa3.png)

**Comment by discordier** [comment]
It seems to be a plain MetaModel without variants and tree mode?

Could you shed some light on the configuration?

**Comment by cboelter** [comment]
Yes it is a plain metamodels with nothing special like variants or treemode.

![bildschirmfoto 2015-02-05 um 15 16 28](https://cloud.githubusercontent.com/assets/550207/6061696/06143122-ad4a-11e4-93a3-71c3c7c415a1.png)
![bildschirmfoto 2015-02-05 um 15 15 58](https://cloud.githubusercontent.com/assets/550207/6061697/061873b8-ad4a-11e4-9748-f6b3a81a3fa3.png)

Do you need more Information. MetaModels is on the current commit and dcgeneral is using the 2.0.X-Branch.

**Comment by psren** [comment]
Maybe related to contao-community-alliance/dc-general#84

I am sure that i had this issue, too. I didn't write a ticket for some reason and now i cannot reproduce it.

**Comment by cboelter** [comment]
@discordier If you want, I can send login data for the project.

**Comment by Schnubl** [comment]
Have the same issues!

Contao 3.3.5
MM Latest Nightly dev-tng
DC-General 2.0.0-beta22

**Comment by cyberlussi** [comment]
Same for me. Plain simple flat MM, no variants.

contao/core 	3.2.20
contao-community-alliance/dc-general 	dev-develop (02c33c05)
metamodels/core 	dev-tng (435456af)

**Comment by MiniModel** [comment]
This issue has been labeled with "testing" but got no feedback.

We are reliant on feedback to prove a ticket fixed as we can not tell if the ticket has been
resolved if we receive no feedback.
I will close this ticket now, as I assume it has been resolved.
If the ticket should still be valid, please feel free to reopen it.

@cboelter could you please have a final look at this and tell if your issue has been resolved?

**Comment by cyberlussi** [comment]
Hi,
Great a fix for that issue! Unfortunately the fix does only works for "manual sorting", once you sort by a field e.g. date, no paste button appears.

**Comment by discordier** [comment]
@dmolineus Can you have a furthe look into this?

**Comment by dmolineus** [comment]
@discordier I'm working on it.

**Comment by cyberlussi** [comment]
:+1:

**Comment by dmolineus** [comment]
@discordier Can get in the testing phase. Fixed via https://github.com/contao-community-alliance/dc-general/issues/134.

**Comment by cyberlussi** [comment]
contao-community-alliance/dc-general 	dev-develop (388f7a7c)
metamodels/core 	dev-master (73c63624)

okay, copying an entry is working, 99% perfect.

1) copy of entry with fixed sorting (e.g. date) is okay for me: Once I clicked the "+", the new entry is already copied and the edit view is opened. The first time it was not obvious for me (do I edit the orginating entry or a new one?), but once you used it several times it is fine. 

2) copy of manual sorted entry is basically working, but not as I expected (Contao style). Clicking the "+" creates the attached view. But instead of selecting already the target sorting position I am forced to do 2 steps (click to topmost "paste" symbol and then resort) as the other "paste here" are greyed out.

![2015-08-13_16_36_04_-_contao_open_source_cms_3_5](https://cloud.githubusercontent.com/assets/4976687/9252834/ce96de72-41da-11e5-84f4-d7f2f92eaad4.png)

**Comment by asaage** [comment]
Copy works fine with fixed sorting.
works (basically) with manual sorting as stated by @cyberlussi.
not shure about copy with no sorting defined...

**Comment by dmolineus** [comment]
@cyberlussi @asaage Thanks for the feedback. I know that I stumpled upon it lately. As far as I remember this issue is addressed by https://github.com/MetaModels/core/pull/856 as well. But it depends on #151 which is on hold so far because some unit tests has to be completed.

**Comment by asaage** [comment]
also:
do not show the blue move arrow in flat mode as there is the drag-handle.

**Comment by MiniModel** [comment]
This issue has been labeled with "testing" but got no feedback.

We are reliant on feedback to prove a ticket fixed as we can not tell if the ticket has been
resolved if we receive no feedback.
I will close this ticket now, as I assume it has been resolved.
If the ticket should still be valid, please feel free to reopen it.

@cboelter could you please have a final look at this and tell if your issue has been resolved?

**Comment by cyberlussi** [comment]
Unchanged behavior compared to my last statement (https://github.com/MetaModels/core/issues/743#issuecomment-130715272).

metamodels/core 	dev-master (3b171dcf)
contao-community-alliance/dc-general 	dev-develop (388f7a7c)
